src/main/antlr/OpenDistroSqlParser.g4

```diff
@@ -413,6 +413,7 @@ functionNameBase
     | E | EXP | EXPM1 | FLOOR | LOG | LOG10 | LOG2 | LOWER
     | PI | POW | RADIANS | RANDOM | RINT | ROUND
     | SIN | SINH | SQRT | SUBSTRING | TAN | TRIM | UPPER | YEAR
+    | IF
```

Comment:
any special reason not to add `IFNULL  and `ISNULL` here?

Reply:
Nope, I've not yet updated the code changes from the local repository thoroughly, I push the updates by today.